dnsimple/registrar_delegation.go

```diff
@@ -7,12 +7,28 @@ import (
 // Domain represents a list of name servers that correspond to a domain delegation.
 type Delegation []string
 
+// Vanity Name Server
+type VanityNameServer struct {
```

Comment:
Actually, I've just noticed that we don't use a struct but a simple `[]string` for standard name servers. At this point, I would probably keep this code as it is.